docs/install/index.md

```diff
@@ -72,9 +72,13 @@ dotnet add package Microsoft.AI.MachineLearning
 
 ## Install on web and mobile
 
-The installation instructions in this section use pre-built packages that include support for selected operators and ONNX opset versions based on the requirements of popular models. Your model must only use the [opsets and operators](../reference/operators/mobile_package_op_type_support_1.9.md) supported by the pre-built package.
+Unless stated otherwise, the installation instructions in this section refer to pre-built packages that include support for selected operators and ONNX opset versions based on the requirements of popular models. These packages may be referred to as "mobile packages". If you use mobile packages, your model must only use the supported [opsets and operators](../reference/operators/mobile_package_op_type_support_1.9.md).
 
-If the pre-built package is too large, or does not include the operators in your model/s, you can create a [custom build](../build/custom.md).
+Another type of pre-built package has full support for all ONNX opsets and operators, at the cost of larger binary size. These packages are referred to as "full packages".
+
+If the pre-built mobile package is too large, you can create a [custom build](../build/custom.md). A custom build can contain just the opsets and operators that you need, so it may be smaller.
+
```

Comment:
nit: It should definitely be smaller so 'may be smaller' doesn't quite convey that it's worth spending the time on the custom build.

For it to be larger, you would need to use more operators/opsets/types than the pre-built package - which has a config based on ~60 models and supports 4 different onnx opsets.